src/pages/Hojung/Main/Main.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import './Main.scss';
+import MainFeed from './MainFeed';
+import SideMenu from './SideMenu';
 
 class MainHojung extends React.Component {
+  logout = () => {
+    localStorage.removeItem('token');
+    this.props.history.push('/login-hojung');
+  };
   render() {
-    return <div>hi</div>;
+    const username = this.props.history.location.state.username;
+    console.log(username);
+    return (
```

Comment:
```suggestion
    const { username } = this.props.history.location.state;
    
    return (
```
- console.log 지워주세요
- 구조분해 할당 사용해주실 수 있을거 같아요!

Reply:
수정완료 했습니다